Update Handbook for Changes to Checkout (#912)
carpentries/docs.carpentries.org: topic_folders/instructor_training/email_templates_trainers.md

According to the [Teaching Demo Etherpad](https://pad.carpentries.org/teaching-demos) you have signed up to give an online teaching demo on [ date ] at [ time ] (See this link for your local time: [ timeanddate.com link ]). I will be the Trainer running the session.

> Please be prepare to teach 1 section (episode) of your chosen Carpentries Lesson. For recommendedations on which lesson preparation, please see the page on [Suggested Lessons for Teaching Demonstrations](https://carpentries.github.io/instructor-training/demo_lessons).

Reviewer: ```suggestion
Please be prepared to teach for 5 minutes, beginning at the start of your chosen episode from a Carpentries lesson. Please review the page on [Suggested Lessons for Teaching Demonstrations](https://carpentries.github.io/instructor-training/demo_lessons) to ensure that your selection is on the list.
```

I’ve forwarded this information to our Core Team. If this was the last step in your Instructor Training checkout, you should hear from them within about a week. 

> Looking forward to having you as a Carpentries Instructor!

Reviewer: I have been trying to reduce words like "looking" where reasonable alternatives are less ableist. In this case I think the sentence can be deleted or left as-is, assuming Trainers will customize anyway.
```suggestion

```

> Looking forward to having you as a Carpentries Instructor!

Reviewer: ```suggestion
Congratulations!
```
